components/core-dependencies/NServiceBus.AmazonSQS.txt

```diff
@@ -1,3 +1,4 @@
+7.0 : 9
```

Comment:
Should we consider a integrity test to check that we don't miss this in the future?